homeassistant/components/flux_led/light.py

```diff
@@ -298,8 +305,10 @@ def turn_on(self, **kwargs):
                 brightness = self.brightness
             if color_temp > COLOR_TEMP_WARM_VS_COLD_WHITE_CUT_OFF:
                 self._bulb.setRgbw(w=brightness)
+                flux_led_wait()
```

Comment:
How is this improving things? Isn't the command done when we call `setRgbw` ?

Reply:
You can check here : https://github.com/beville/flux_led/blob/master/flux_led/__main__.py
setRgbw does nothing more than sending bytes to a socket: no waiting